docs/features/search/search-engines.md

```diff
@@ -144,7 +149,7 @@ const client = searchEngine.newClient(options => {
 
 #### Set custom index template
 
-The elasticsearch engine gives you the ability to set a custom index template if needed.
```

Comment:
is it worth updating https://github.com/backstage/backstage/blob/master/.github/vale/Vocab/Backstage/accept.txt#L102 ?

Reply:
It's a good thought!  But unfortunately, that list appears to be case-insensitive.